src/css/style.css

```diff
@@ -51,6 +51,15 @@
   margin: 25px;
 }
 
+.card:hover {
+  margin-top: 0px;
+  margin-bottom: 50px;
```

Comment:
Instead of animating the margins, you can use `transform` and adjust the shadow accordingly.